### DISCUSSION TITLE
How do get GasPrices?

### DISCUSSION THREAD

**Discussion by 0xZonh** [cosmos/community#43] [Announcements]
Normally, we use gasPrices of 0.025uatom to initiate transactions, but I want to pay as little as possible. I know that the node has a limit of min-gas-prices, so how do we know the min-gas-prices of the node, or how do we estimate the minimum transaction fee?

**Comment by julienrbrt** [comment]
Answered here: https://github.com/cosmos/cosmos-sdk/discussions/17567
